snippets/average.md

```diff
@@ -6,7 +6,7 @@ function average(...$items)
 {
     $count = count($items);
     
-    return $count === 0 ? 0 : array_sum($items) / $count;
+    return $count === 0 ? NULL : array_sum($items) / $count;
```

Comment:
if it's the empty array `items`, it should be the `null`, not `0`.

It can be more current than original appraoch.

Could you let the `NULL` be lowercase? I think it should be consistent for the whole source codes.